Content.Server/GameObjects/Components/GUI/ServerHandsComponent.cs

```diff
@@ -406,6 +406,11 @@ public bool HasHand(string index)
             return hands.ContainsKey(index);
         }
 
+        public void RefreshInHands()
```

Comment:
This feels like something that should never be called directly on the server. The item netcode should take care of it automatically.

Reply:
Alright, I made this update on a new component state instead